samples/snippets/src/main/java/com/example/video/Detect.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 Google Inc.
+ * Copyright 2017 Google Inc.
```

Comment:
Could you check this?  They don't usually go backwards, typically it doesn't ever change.